src/SpecterWallet.ts

```diff
@@ -8,6 +8,9 @@ import { getOnChainTransactions } from "./OnChain"
 import { UserWallet } from "./userWallet"
 import { BitcoindClient, bitcoindDefaultClient, btc2sat, sat2btc } from "./utils"
 
+// TODO: Centralizing this special handling. Does not seem to be the "outside" here...
+const specialWalletName = "default" // was ""
```

Comment:
Why is it being called `specialWallet`? Perhaps we can find a better (more descriptive) name

Reply:
Agree, that is a temporary name until I am more certain about how to call it. Also, maybe to reconsider how it is used.